.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2025-01-20 - [Evolution API Path Traversal]
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)